garble.py

```diff
@@ -70,7 +74,20 @@ def validate_clks(clk_files, metadata_file):
 
 def garble_pii(args):
     secret_file = Path(args.secretfile)
-    source_file = Path(args.sourcefile)
+
+    if args.sourcefile:
+        source_file = Path(args.sourcefile)
+    else:
+        filenames = list(
+            filter(lambda x: "pii" in x and len(x) == 23, os.listdir("temp-data"))
```

Comment:
It looks like this code is related to making `sourcefile` optional. If I'm reading this right, pii files in the temp directory are discovered by selecting files with the string `pii` in the filename and which are 23 characters in length.

Won't the length check break if the timestamp format is changed to one of a different length?

Reply:
I fixed it so now it gets the proper timestamp length from definitions.py